### ISSUE TITLE
Could you please add the code about the net structure and layer define ?(such as triplet loss layer)

### ISSUE THREAD

**Issue by AlbertDu** [davidsandberg/facenet#272]
As said above in the title, thanks!
(now in the models.inception_resnet_v1.py doesn't include the triplet loss layer),  we need help about the details when we train our own model or just validate you model on the certain dataset,
thanks a lot for you work again.

**Comment by davidsandberg** [comment]
The loss function is added in the training script (train_softmax or train_tripletloss). The triplet loss is defined in facenet.py (`triplet_loss(anchor, positive, negative, alpha)`).

**Comment by AlbertDu** [comment]
Thanks a lot for your enthusiastic help !

Best wishes！

I've closed the issue.
